be/src/Infrastructure/Localization/Foundation.Localization/Resources/GlobalResources.pl.resx

```diff
@@ -570,4 +570,7 @@
   <data name="VicePresident" xml:space="preserve">
     <value>Wiceprezes</value>
   </data>
+  <data name="AcceptTermsText" xml:space="preserve">
+    <value>Akceptuję Regulamin i Politykę Prywatności</value>
```

Comment:
Hi @SebaKas21 ,

let's make Terms and Conditions & Privacy Policy links that lead to the terms and conditions and privacy policy pages.

Cheers,